src/react.tsx

```diff
@@ -92,3 +92,88 @@ export function connectAs<
     }
   }
 }
+
+type IdentityWith<A> = (a: A) => A
+let identity: IdentityWith<any> = _ => _
+
+export function createStoreStrict<State extends object>(
```

Comment:
This will probably just be `createStore`, and I'll kill the existing `createStore` and `connect` functions.